tests/server-integration/Integration.hs

```diff
@@ -0,0 +1,122 @@
+-- SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
+--
+-- SPDX-License-Identifier: MPL-2.0
+
+module Integration where
+
+import Control.Monad (void)
+import Data.Aeson (Value(Object), (.:))
+import Data.Aeson.QQ.Simple (aesonQQ)
+import Data.Aeson.Types (parseEither)
+import Data.Functor ((<&>))
+import Data.Text (Text)
+import Network.HTTP.Req
+import Test.Tasty.HUnit
+import Utils
+
+unit_create_an_entry :: IO ()
+unit_create_an_entry = cofferTest do
+  response <-
+    runReq defaultHttpConfig do
+      req
+        POST
+        (http "localhost" /: "api" /: "v1" /: "content" /: "create")
```

Comment:
Seems lines 23, 58, 70 have same configs except one argument. So wouldn't it be better to make presetted config for them

Reply:
Done